### DISCUSSION TITLE
Can't execute arm-estimator on MacOs

### DISCUSSION THREAD

**Discussion by Glendeschema** [TheCloudTheory/arm-estimator#71] [General]
Hi there ,

I am attempting to run the arm-estimator from MACOS and seem to have difficutly.

I have installed latest bicep CLI and az CLI.

Glens-MacBook-Pro:arm-estimator-linux-x64-1 glen$ dotnet arm-estimator template-path /Users/glen/Bicep_lab/multiplevms.bicep  subscription-id 07f0d279-bad1-4780-bf65-d638a02xxxx resource-group glen2 --mode Incremental
Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET Core program, but dotnet-arm-estimator does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.

**Comment by kamil-mrzyglod** [comment]
Hey @Glendeschema, is it possible you misused parameters syntax? Could you use the following:
```
./arm-estimator /Users/glen/Bicep_lab/multiplevms.bicep 07f0d279-bad1-4780-bf65-d638a02xxxx glen2 --mode Incremental
```

In details:
* don't prepend `dotnet` and instead execute `arm-estimator` as any other application
* do not use named parameters and instead just pass values in the following order: <template-path> <subscription-id> <resource-group-name>

Could you also confirm you're using `beta2` version?
